ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/user_management_service.proto

```diff
@@ -189,18 +207,22 @@ message UpdateUserResponse {
     User user = 1;
 }
 
-// Required authorization: ``HasRight(ParticipantAdmin)``
+// Required authorization: ``HasRight(ParticipantAdmin) OR IsAuthenticatedIdentityProviderAdmin(identity_provider_id)``
 message DeleteUserRequest {
     // The user to delete.
     // Required
     string user_id = 1;
+
+    // The id of the ``Identity Provider``
+    // Optional, if not set, assume the user is managed by the default identity provider.
```

Comment:
So if I'm an idp admin of idp 'foo' I always need to set this field to 'foo' because otherwise I'd be attempting to delete a user belonging the default idp config (the one configured statically at the participant deployment time) and would get an user_not_found or access_denied error?

Reply:
access_denied, because you cannot use default idp if you're admin of "foo".